java/api/src/main/java/org/ray/api/PyActorCall.java

```diff
@@ -2,40 +2,42 @@
 
 package org.ray.api;
 
+import org.ray.api.function.PyActorMethod;
+
 /**
  * This class provides type-safe interfaces for remote actor calls.
  **/
 @SuppressWarnings({"rawtypes", "unchecked"})
 interface PyActorCall {
 
-  default RayObject call(String functionName) {
+  default RayObject call(PyActorMethod pyActorMethod) {
```

Comment:
```suggestion
  default <R> RayObject<R> call(PyActorMethod<R> pyActorMethod) {
```

`Ray.call` methods too.

Reply:
PyCallGenerator.java has updated for generic type.